apps/bfd-model/bfd-model-rif/src/main/java/gov/cms/bfd/model/rif/parse/RifParsingUtils.java

```diff
@@ -178,12 +188,8 @@ public static Optional<Long> parseOptionalLong(String longText) {
    * @return the specified text parsed into a {@link BigDecimal}
    */
   public static BigDecimal parseDecimal(String decimalText) {
-    /*
-     * Might seem silly to pull this out, but it makes the code a bit easier
-     * to read, and ensures that this parsing is standardized.
-     */
     if (decimalText.isEmpty()) {
-      return new BigDecimal(0);
+      return BigDecimal.valueOf(0);
```

Comment:
🍰  Can also use BigDecimal.ZERO

Reply:
good call (and more efficient). changing to use .ZERO.